### PR TITLE
Switch to Travis container arch and test MRI 2.2.3

### PR DESCRIPTION
**Pull request by rymai**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 rvm:
   - 1.9.3
   - 2.0.0
+  - 2.1.7
+  - 2.2.3
```
Comment (justincampbell) [inline .travis.yml]: I'd suggest making these 2.1 and 2.2 (dropping the patch version).

**Comment by justincampbell** [comment]
:+1: pending comment above

**Comment by rymai** [comment]
Here you go, thanks!

**Comment by justincampbell** [comment]
:+1: :heart: :100:

**Pull request merged**